.github/scripts/codespell.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
```

Comment:
pls add copyright and license (similarly to https://github.com/oneapi-src/unified-memory-framework/blob/main/.github/scripts/get_system_info.sh#L4 )

Reply:
Done